dotnet/src/Plugins/Plugins.UnitTests/Web/Google/GoogleTextSearchTests.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Google.Apis.CustomSearchAPI.v1.Data;
+using Google.Apis.Http;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Plugins.Web.Google;
+using Microsoft.SemanticKernel.Search;
+using Xunit;
+
+namespace SemanticKernel.Plugins.UnitTests.Web.Google;
+
+public sealed class GoogleTextSearchTests : IDisposable
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="GoogleTextSearchTests"/> class.
+    /// </summary>
+    public GoogleTextSearchTests()
+    {
+        this._messageHandlerStub = new MultipleHttpMessageHandlerStub();
+        this._clientFactory = new CustomHttpClientFactory(this._messageHandlerStub);
+        this._kernel = new Kernel();
+    }
+
+    /// <inheritdoc/>
+    public void Dispose()
+    {
+        this._messageHandlerStub.Dispose();
+        this._clientFactory.Dispose();
+
+        GC.SuppressFinalize(this);
+    }
```

Comment:
Move this to the end have the tests as close as possible to the beggining of the class.